### PR TITLE
Add raw-content plugin, get raw embed description content

### PR DESCRIPTION
**Pull request by Nzii3**

### CODE REVIEW

**Comment by Nzii3** [comment]
For more context, you can’t copy embed description content on iOS unlike Android or other mobile devices.

**Comment by fourjr** [comment]
Please update to v4 first.

**Comment by StephenDaDev** [comment]
@Nzii3 Not sure if you still want to add this in or not

**Comment by Nzii3** [comment]
> Please update to v4 first.

Done

**Comment by Nzii3** [comment]
> @Nzii3 Not sure if you still want to add this in or not

Yeah I do.

File: plugins/registry.json
Comment (StephenDaDev) [inline plugins/registry.json:1]: Did not provide a thumbnail or logo image, please provide something to keep consistency with every other plugin in the registry

**Review by StephenDaDev — CHANGES_REQUESTED** [review]

File: plugins/registry.json [hunk 1]
```diff
@@ -255,5 +255,11 @@
         "title": "Scam Link Detector",
         "icon_url": "https://cdn.discordapp.com/attachments/576521645540245505/895661244743299102/antifish.png",
         "thumbnail_url": "https://cdn.discordapp.com/attachments/576521645540245505/895661244743299102/antifish.png"
+    },
+    "raw-content": {
+        "repository": "Nzii3/modmail-plugins",
+        "branch": "main",
+        "description": "Have the ability to get raw embed description (useful for iOS mobile users).",
+        "title": "Raw Content",
```
Comment (Taaku18) [inline plugins/registry.json:263]: As mentioned by Stephen, it's preferred to include an icon and thumbnail (given by URL). Also, I don't believe you're allowed to end JSON objects with a trailing comma, that would probably trigger a parsing error.

**Review by Taaku18 — CHANGES_REQUESTED** [review]

File: plugins/registry.json (hunk 1 above)
Comment (Taaku18) [reply plugins/registry.json]: I've fixed the comma issue, but would still prefer to have the two images.

**Comment by Nzii3** [comment]
I don’t have a thumbnail for it

**Comment by Taaku18** [comment]
> I don’t have a thumbnail for it

Feel free to find one from the internet (preferably not copyrighted), or you can simply use the Modmail icon as some other plugins have done (https://i.imgur.com/Mo60CdK.png).

**Comment by StephenDaDev** [comment]
@Nzii3 Please make these changes if you wish to proceed with this.

**Pull request closed without merging**